### PR TITLE
Refactor to use java script classes

### PR DESCRIPTION
**Pull request by samuelmoiwa**
## Solution description

**CSS file:** update CSS style
**HTML file:** multiple dives boxes added in index .HTML to separate files and enhance UI design
**JavaScript file:** implement a class for local storage in main.js
**Linters files:** linters files configured

## Address affected and ensured
* HTML file
* CSS file
* JavaScript
* Linters Config
---

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -10,46 +11,43 @@ const clear = () => {
   author.value = '';
 };
```
Comment (KDlamini) [inline main.js]: On line 7 you have your books array outside of the class

image

- Kindly make sure that the use of objects and arrays is **inside** the classes.

Your class structure should be similar to this:

image

File: main.js [hunk 2]
```diff
@@ -63,7 +61,7 @@ const addBook = (element) => {
```
Comment (KDlamini) [inline main.js]: On line 55 you have your `add book` method outside of the class.

image

- Kindly make sure that your add book method is inside the class as demonstrated in the comment above.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi team @samuelmoiwa, @AbayomiOlaoye 🙋‍♀️,  

Great work so far! 💯 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Correct PR title and description
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js [hunk 3]
```diff
@@ -1,74 +1,78 @@
...
+    const addedBook = {
+      title: title.value,
+      author: author.value,
+    };
+
+    this.availableBooks.push(addedBook);
+    clear();
+    ClassLocalStorage.saveToLocalStorage(this.availableBooks);
+    displayItem();
+  };
```
Comment (Wes-Isaac) [inline main.js]: - [x] Great job so far, when I try to add books it's giving me this error, kindly fix that.

Screen Shot 2022-09-22 at 9 14 02 AM

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @samuelmoiwa & @AbayomiOlaoye, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights
- No linter error ✅ 
- Professional README.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: main.js (hunk 3 above)
Comment (samuelmoiwa) [reply main.js]: hello, thanks for the review. 
it actually works fine for me with no error.

Comment (samuelmoiwa) [reply main.js]: please recheck the project. I think the error might be on your part, as it works well for me. have rerun the project several times.
books ss

**Review by aimemalaika — APPROVED** [review]
Hi @samuelmoiwa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉

- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**